content/c/concepts/linked-list/linked-list.md

```diff
@@ -0,0 +1,325 @@
+---
+Title: 'Linked List'
+Description: 'A data structure that contains a collection of nodes that together with the help of pointers, display a sequence.'
+Subjects:
+  - 'List Types'
+  - 'Computer Science'
+Tags:
+  - 'Singly linked lists'
+  - 'Doubly linked lists'  
+  - 'Circular linked lists'
+  - 'Circular doubly linked lists'
```

Comment:
Here apart from `Double Linked List` the other tags don't exist in the `tags.md` file. Please add them there or remove these tags from here and add one that is present in the `tags.md` file.

Reply:
Hi @dakshdeepHERE I am having trouble understanding your request. When you say, add the tags to a tags.md file do you mean create a tags.md file?
Or are you saying to edit something within the linked-list.md file?

Reply:
@Carloscode5 the `tags` should come from [here](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) only i.e. `tags.md`. If the tags you used are not there then go to `tags.md` and add the tags that you've used here in the file. Sorry if I was not clear before @Carloscode5. Is there anything else I can help you with?